src/Client/TrapHandle/Dumper.php

```diff
@@ -132,7 +133,7 @@ private static function getContextProviders(): array
 
         return $contextProviders + [
             'cli' => new CliContextProvider(),
-            'source' => new ContextProvider\Source(null, null, $fileLinkFormatter),
+            'source' => new Source(null, null, $fileLinkFormatter),
```

Comment:
Don't replace relative namespaces

Reply:
Why not? In this file the specified class is used only once. There are no other places where the root namespace used.

Reply:
This practice doesn't apply well to Trap because relative names add context. We don't have Interface or ContextProvider suffixes in class and interface names here. This context is taken from the namespace. And if the programmer decides that a relative name is better, the rector should not interfere with that